lib/ansible/modules/pip.py

```diff
@@ -306,6 +307,18 @@ def _is_vcs_url(name):
     return re.match(_VCS_RE, name)
 
 
+def _is_venv_command(command):
+    venv_parser = argparse.ArgumentParser()
+    venv_parser.add_argument('-m', type=str)
```

Comment:
I wonder if we should move the `venv_parser` definition into the global scope in case the function is used more than once. But it is used just once now, not sure about the likelihood of it changing, so up to you.

Reply:
Yeah, I'm not sure where it can be used either - virtualenv sounds quite relative to the pip module.